.github/workflows/main.yml

```diff
@@ -3,7 +3,10 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-10.15,  ubuntu-20.04]
```

Comment:
So easy! What other OS flavors are available?

Reply:
Ubuntu 16, 18, 20

macOS 10.15, 11.0

Windows Server

Reply:
Seems like we should use all of them, except Windows.